packages/iov-tendermint-rpc/src/v0-20/requests.ts

```diff
@@ -31,6 +32,16 @@ export class Params extends requests.DefaultParams {
     return jsonRpcWith(req.method, req.params);
   }
 
+  public static encodeSubscribe(req: requests.SubscribeRequest): JsonRpcRequest {
+    const allTags: ReadonlyArray<Tag> = [
+      { key: "tm.event", value: req.query.type },
+      ...(req.query.tags ? req.query.tags : []),
```

Comment:
Style question...

In such a case, i would usually write `...(req.query.tags || [])`
Both versions are fine. Wondering if there is a reason to use one over the other, or just personal preference.

Reply:
Both ways are fine. The ternary operator was the best thing that case to my mind.

I slightly prefer to keep the test and the return value independent allowing modifications to one or the other more easily, e.g. `(req.query.tags === null ? req.query.tags : [])`